File: src/test/java/application/history/endpoint/history/api/DeleteHistoryTest.java
package application.history.endpoint.history.api;

import application.history.IntegrationTest;
import application.history.WithFaker;
import application.history.history.application.services.DeleteHistoryService;
import lombok.SneakyThrows;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;


public class DeleteHistoryTest extends IntegrationTest implements WithFaker {

    @Autowired
    private DeleteHistoryService service;

    @SneakyThrows
    @Test
    public void deleteHistory() {

    }
}

File: src/history/test/kotlin/com/example/history/application/history/endpoint/history/api/DeleteHistoryTest.kt
package com.example.history.application.history.endpoint.history.api

import application.history.IntegrationTest
import org.junit.jupiter.api.Test

class DeleteHistoryTest : IntegrationTest(), WithFaker {
    @Autowired
    private val service: DeleteHistoryService? = null
    @SneakyThrows
    @Test
    fun deleteHistory() {
    }
}
